apps/expo-go/src/utils/UrlUtils.ts

```diff
@@ -1,9 +1,20 @@
+import { createRuntimeUrl } from 'snack-content';
 import url from 'url';
 
+import Environment from './Environment';
 import Config from '../api/Config';
 
 const HTTPS_HOSTS = [Config.api.host, 'exp.host', 'exponentjs.com', 'getexponent.com'];
 
+export function normalizeSnackUrl(fullName: string, channelName?: string): string {
+  return createRuntimeUrl({
+    // Only ever try to open Snack using the current SDK version
```

Comment:
No worries if you don't do this btw, basically users will run into the "This SDK version is not supported" screen. So it's still a useful/actionable error.